torch/csrc/jit/pybind_utils.h

```diff
@@ -26,22 +26,22 @@ inline void findErrorInKwargs(
   // any argument in the schema.
   for (const auto& kwarg : kwargs) {
     const auto key = py::cast<std::string>(kwarg.first);
-    AT_CHECK(
-        std::count_if(
+    if(!std::count_if(
             arguments.begin(),
             arguments.end(),
-            [&key](const Argument& argument) { return argument.name == key; }),
-        "Unknown keyword argument '", key, "' for operator '",
-        schema.name, "'. Schema: ", schema);
+            [&key](const Argument& argument) { return argument.name == key; })) {
+        throw std::runtime_error(at::str("Unknown keyword argument '", key, "' for operator '",
+        schema.name, "'. Schema: ", schema));
```

Comment:
If the problem here was with `AT_CHECK`, then we should probably add some comments. Otherwise, sooner or later, someone will codemod those to become `AT_*` macros again. 😕 Ideally we'd modify pybind to know that it shouldn't display those messages. `HANDLE_TH_ERRORS` macros disable them.